### PR TITLE
Add Parallel Letter Frequency exercise

### PR DESCRIPTION
**Pull request by kahgoh**

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -694,6 +694,14 @@
...
         "prerequisites": [],
         "difficulty": 4
+      },
+      {
+        "slug": "parallel-letter-frequency",
+        "name": "Parallel Letter Frequency",
+        "uuid": "61b5fc26-dccd-4ed2-a15e-723273632d5d",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 4
```
Comment (1ethanhansen) [inline config.json:704]:
```suggestion
        "difficulty": 5
```

I might bump this up to a 5, what do you think?

**Review by 1ethanhansen — COMMENTED** [review]
Just thinking about the difficulty rating, otherwise LGTM

File: config.json (hunk 1 above)
Comment (kahgoh) [reply config.json:704]: I think going up to 5 is reasonable, especially because of the Unicode test case.

**Review by 1ethanhansen — APPROVED** [review]
thank you!

**Pull request merged**